.github/workflows/build.yml

```diff
@@ -62,7 +62,9 @@ jobs:
           java-version: 11
 
       - name: Publish Artifacts
-        run: ./gradlew publishMavenPublicationToMavenCentralRepository paparazzi-gradle-plugin:publish
+        run: |
+          ./gradlew publishMavenPublicationToMavenCentralRepository paparazzi-gradle-plugin:publish
+          ./gradlew publishMavenPublicationToMavenCentralRepository paparazzi-accessibility:publish
```

Comment:
@jrodbx Not as familure with the maven publish plugin we are using but I added this to make sure the artifact is also published